architect_py/client.py

```diff
@@ -1,26 +1,58 @@
-from typing import Literal
+from datetime import datetime, timezone
+from decimal import Decimal
+from typing import Literal, Optional
+
+from architect_py.graphql_client.enums import CreateOrderType, OrderSource
 from .graphql_client import GraphQLClient
-from .graphql_client.input_types import CreateOrder, CreateTimeInForce
+from .graphql_client.input_types import (
+    CreateOrder,
+    CreateTimeInForce,
+    CreateTimeInForceInstruction,
+)
 
 
 class Client(GraphQLClient):
     async def send_limit_order(
         self,
+        *,
```

Comment:
what does the * do here? i need a reminder lol

Reply:
it forces someone using the function to pass arguments by keyword
e.g.
instead of `send_limit_order(market, side, ...)` it will need to be
`send_limit_order(market=market, side=side, ...)`

imo less room for catastrophic bugs for someone using the API and more room for changes to this function in case we make some changes later